test/Base_Tests/src/Data/Polyglot_Spec.enso

```diff
@@ -42,6 +42,26 @@ add_specs suite_builder = suite_builder.group "Polyglot" group_builder->
         hold.value . should_equal 42
         hold.boxed . should_equal 42
 
+    group_builder.specify "extension method on IntHolder" <|
+        hold = IntHolder.new (6 * 7)
+        v = hold.value_plus 3
+        v . should_equal 45
+
+    group_builder.specify "extension method on IntHolder (static syntax)" <|
+        hold = IntHolder.new (6 * 7)
+        v = IntHolder.value_plus hold 3
```

Comment:
With the latest commit af736be149fbb75ec8f57c92c8a1a5a9aa469b72 all tests but this one seem to be passing:
```
    - [FAILED] extension method on IntHolder (static syntax) [98ms]
        Reason: An unexpected panic was thrown: (Type_Error.Error Integer org.enso.base_test_helpers.IntHolder@16120270 '`add`')
        at <enso> Any.value_plus(/enso/test/Base_Tests/src/Data/Polyglot_Spec.enso:78:1-54)
        at <enso> <anonymous><arg-1>(enso/test/Base_Tests/src/Data/Polyglot_Spec.enso:52:13-39)
```
for some reason I have an oversaturated argument there. Any clue, @hubertp? I guess you must have seen this when implementing the _static syntax_ invocation.